duckduckgo/macos-browser
# fix outlook popups loading; set popup window mode to match Safari

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204100844398175/f

**Description**:
- Fixes popups rendering in Outlook
- Changed `window.open` window mode choosing to match Safari

**Steps to test this PR**:
Prepare a test html page with content:
```
<html><body>
<br /><br /><br />
<a href='javascript:window.open("https://duckduckgo.com","_blank")'>popup _blank</a>
<br /><br /><br />
<a href='javascript:window.open("https://duckduckgo.com","_blank","popup=1")'>popup popup=1</a>
<br /><br /><br />
<a href='javascript:window.open("https://duckduckgo.com","_blank","popup=1, width=320, height=480, left=2000, top=500")'>popup with width/height</a>
</html></body>
```

1. Open the test page, compare the links behaviour to Safari - opened kind of Tab/Window/Popup should match
2. Open outlook.com, open a mail, wait until it fully loads
3. Click sender link, click one of the "recent talks" tiles - popup should open and load (see the asana task for details on these steps to reproduce)
4. Click a link in the popup - new window should open
5. When in full screen mode - all popups should be open as new tabs



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
